app/mailers/user_mailer.rb

```diff
@@ -317,15 +317,17 @@ def in_person_ready_to_verify(enrollment:, is_enhanced_ipp:)
     end
   end
 
-  def in_person_ready_to_verify_reminder(enrollment:)
+  def in_person_ready_to_verify_reminder(enrollment:, is_enhanced_ipp:)
```

Comment:
Does `is_enhanced_ipp` have to be passed in as a separate parameter? Why not use the `enrollment.is_enhanced_ipp?` method?

Reply:
I'll try that. 👍🏻